pos_payment_terminal/models/account_journal.py

```diff
@@ -0,0 +1,13 @@
+# © 2014-2016 Aurélien DUMAINE
+# © 2015-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    payment_mode = fields.Selection(
+        [('card', 'Card'), ('check', 'Check')], 'Payment Mode',
+        help="Select the payment mode sent to the payment terminal")
```

Comment:
There were some interesting comments by @chienandalu and @rousseldenis in https://github.com/OCA/pos/pull/301 about either renaming this to `pos_terminal_payment_mode` or integrating it with `payment.mode` model

https://github.com/OCA/pos/pull/301#discussion_r277226621

Maybe it's worth exploring it? What do you think?

Reply:
Hi. 
thanks for pointing this comments. 
- for the time being, I'm in favor to rename the field as proposed by @chienandalu.

- The proposal of @rousseldenis is not valid I think, because this field has nothing to do with payment mode of the module ``account_payment_mode`` module. (except the name ;-)). The value of the selection fields only depends on the hardware you have. (credit cart terminal, or check printer).

Reply:
Sounds good to me :+1: